pytorch_lightning/utilities/enums.py

```diff
@@ -254,8 +254,6 @@ def interactive_compatible_types() -> list[_StrategyType]:
         """Returns a list containing interactive compatible _StrategyTypes."""
         return [
             _StrategyType.DP,
-            _StrategyType.DDP_SPAWN,
-            _StrategyType.DDP_SHARDED_SPAWN,
```

Comment:
@awaelchli It's supported, right?

Reply:
No, pytorch does no longer support multiprocessing in jupyter notebooks and cuda. only fork+cpu seems to be working according to some tests I did weeks ago. #7550

Reply:
added a comment in the code